core/Cargo.toml

```diff
@@ -14,12 +14,13 @@ readme.workspace = true
 bench = false
 
 [features]
-default = ["markdown", "repl", "doc", "format"]
-markdown = ["termimad"]
-repl = ["rustyline", "rustyline-derive", "ansi_term"]
-repl-wasm = ["wasm-bindgen", "js-sys", "serde_repr"]
-doc = ["comrak"]
-format = ["topiary", "topiary-queries", "tree-sitter-nickel"]
+default = ["markdown", "repl", "doc", "format", "metrics"]
+markdown = ["dep:termimad"]
```

Comment:
The cargo docs say that's available since rust 1.60. I needed to use it for the `metrics` feature at some point, because I originally tried using `metrics-util` as well. So I had a feature definition `metrics = ["metrics", "metrics-util"]` which cargo understandably yelled at me for. Using the optional dependency with `dep:` instead doesn't create an implicit feature of the same name as the dependency and makes it a bit more explicit which features are actually intended.

Reply:
I see. It's indeed better that way